src/llmq/params.h

```diff
@@ -193,7 +193,7 @@ static constexpr std::array<LLMQParams, 10> available_llmqs = {
 
         .signingActiveQuorumCount = 2, // just a few ones to allow easier testing
 
-        .keepOldConnections = 3,
+        .keepOldConnections = 5,
```

Comment:
should be `signingActiveQuorumCount * 2` as far as I understand, so I guess it should be
```suggestion
        .keepOldConnections = 4,
```
would be nice to have lines 93-94 updated with the description for rotation quorums

Reply:
I agree, @ogabrielides please add this description in a new PR when you wake up :)